packages/@aws-cdk/aws-msk/lib/cluster.ts

```diff
@@ -616,6 +623,16 @@ export class Cluster extends ClusterBase {
       clientAuthentication = {
         sasl: { iam: { enabled: props.clientAuthentication.saslProps.iam } },
       };
+      if (props.clientAuthentication?.tlsProps) {
+        clientAuthentication = {
+          sasl: { iam: { enabled: props.clientAuthentication.saslProps.iam } },
+          tls: {
+            certificateAuthorityArnList: props.clientAuthentication?.tlsProps?.certificateAuthorities?.map(
+              (ca) => ca.certificateAuthorityArn,
+            ),
+          },
+        };
```

Comment:
You can avoid duplicating code by doing:

```suggestion
        clientAuthentication.tls = {
            certificateAuthorityArnList: props.clientAuthentication?.tlsProps?.certificateAuthorities?.map(
              (ca) => ca.certificateAuthorityArn,
            ),
        };
```

Reply:
Yes thank you